src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    class="ec-summary-info"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-summary-info` : `ec-summary-info`"
+  >
+    <ec-icon
+      v-if="iconName"
+      class="ec-summary-info__main-icon"
+      data-test="ec-summary-info__main-icon"
+      :name="iconName"
+      :size="16"
+    />
+
+    <slot>
+      <div
+        data-test="ec-summary-info__content"
+        class="ec-summary-info__content"
+      >
+        <div
+          v-for="(item, index) in lineItems"
+          :key="item.text"
+          class="ec-summary-info__content-line-item"
```

Comment:
Can we please add here a unique data-test. In theory we can pass multiple "descriptions", "text" etc per summaryinfo component.

If this happens we will not have a way in the tests to target them. Something like the following maybe
```
:data-test="`ec-summary-info__content-line-item-${index}`"
```